ts/Dashboards/Plugins/HighchartsComponent/HighchartsComponent.ts

```diff
@@ -281,6 +281,13 @@ class HighchartsComponent extends Component {
         hcComponent.chart = hcComponent.getChart();
         hcComponent.updateSeries();
 
+        if (!hcComponent.cell.container.style.height) {
```

Comment:
Do we need to put it in the `render`?
It  might be triggered very often and this is one time thing when the connector loads 🤔

Reply:
We have the same solution in the KPI

Reply:
That is why I wonder if there is a better way to do it.